src/it/resources/log4j.properties

```diff
@@ -1 +1,9 @@
-log4j.rootLogger=OFF
+# Set everything to be logged to the console
+log4j.rootCategory=WARN, console
+log4j.appender.console=org.apache.log4j.ConsoleAppender
+log4j.appender.console.target=System.err
+log4j.appender.console.layout=org.apache.log4j.PatternLayout
+log4j.appender.console.layout.ConversionPattern=%d{yy/MM/dd HH:mm:ss} %p %c{1}: %m%n
+
+
+log4j.logger.io.github.spark_redshift_community.spark.redshift=INFO
```

Comment:
logging should be disable by default, as it was before

Reply:
Done. It is now OFF by default.